app/events/talk/page.tsx

```diff
@@ -0,0 +1,84 @@
+          <div className="grid md:grid-cols-2  gap-8">
+            <div className=""><Image src={comTalkImg} alt=''/></div>
+            <div className='flex justify-center items-center mx-auto'>
+              <p className='text-[#C9C3BA] text-[14px]'>
+              Aspiring AI enthusiasts are invited to register for this exceptional journey, delving into four virtual sessions covering OpenAI APIs/Prompt engineering, Retrieval Augmented Generation (RAG) and Fine-tuning LLMs, Training and Deployment strategies and frontend Implementation and Implementing OpenAI Models in Practice. The climax is a thrilling physical hackathon, challenging participants to apply their knowledge to craft innovative solutions to a real-world case study. With a minimal fee for subsequent sessions, the bootcamp ensures access to invaluable resources, mentorship, and the opportunity to showcase solutions.
+
+              Don’t miss this chance to be part of Sri Lanka’s most anticipated AI event, where education meets hands-on experience and creativity flourishes!
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <EventCard className='grid md:grid-cols-2 gap-4 rounded-[0px_40px]'>
+          <div className="flex flex-col gap-4 h-full justify-around md:p-[40px]">
+            <p className='uppercase text-[20px] text-center md:text-left font-semibold tracking-[0.4px]'>april community talk</p>
+            <p className='text-[14px] text-[#C9C3BA] uppercase tracking-[1px] leading-[30px] font-semibold'>Aspiring AI enthusiasts are invited to register for this exceptional journey, delving into four virtual sessions covering OpenAI APIs/Prompt engineering, Retrieval Augmented Generation (RAG) and Fine-tuning LLMs, Training and Deployment...</p>
+            <div className='flex flex-col md:flex-row justify-center items-center md:items-center gap-4'>
+              <div className='flex gap-2'>
+                <Image className='size-[20px]' src={watchIcon} alt=''/>
+                <span>30<sup>th</sup> April 2025</span>
+              </div>
+              <div className='flex gap-2'>
+                <Image className='size-[20px]' src={calendarIcon} alt=''/>
+                <span>6.30 P.M.</span>
+              </div>
+              <div className='flex gap-2'>
+                <span>Via <b>Zoom</b></span>
+              </div>
+            </div>
+          </div>
+          <div><Image src={aprilComTalkImg} alt=''/></div>
+
+        </EventCard>
+
+        <div className='flex flex-col gap-4 p-5'>
+          <div className='uppercase font-elemental-end text-white text-[32px] text-center md:text-left'>
+            what you will <span className='text-primary'>get</span>
+          </div>
+          <div className='grid md:grid-cols-2 gap-4 text-white px-[20px] justify-center items-center'>
+            {eventsFeatsData.map((data) => (
+              <EventFeatCard contactDetail={data}/>
+            ))}
```

Comment:
must use 'key' props with the mapping
```suggestion
            {eventsFeatsData.map((data, i) => (
              <EventFeatCard contactDetail={data} key={i}/>
            ))}
```